tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md

```diff
@@ -0,0 +1,344 @@
+## "Hello World” App - MongoDB Atlas and App Engine Flex - NodeJS
+author: arajwade, ksmith
+tags: App Engine, Node.js, MongoDB Atlas
+date_published: 2018-05-06
+
+## Overview: 
+In this tutorial, we will be building a “Hello World” application using Node JS with Google App Engine Flex for our frontend and a MongoDB Atlas multi-regional cluster on GCP as our primary database.
+
+### Technical Complecity: 
+Beginner
+
+### Duration: 
+45 Minutes
+
+### Objective:
+1. Create and configure MongoDB Atlas multi-regional cluster on GCP
```

Comment:
For this "Objective" list:

- Change to "Objectives"
- Change "our" to "your"
- Add a period to each list item
- Change semicolon in item 5 to a comma